alpha/external-web/app/log4js.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * This file defines a configures the log4js logging library.
+ */
+var log4js = require('log4js')
+
+// TODO: Change the file path to a relative path.
+log4js.configure('/usr/src/app/app/config/log4js-configuration.json')
```

Comment:
Use a environment variable EXT_WEB_LOCATION for path to application, in code use process.env.EXT_WEB_LOCATION || '/usr/src/app'

This will allow people to set the var locally and still get logging if they want